.github/workflows/playwright.yml

```diff
@@ -0,0 +1,44 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [astro, astro-*]
+  pull_request:
+    branches: [astro, astro-*]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: npm
+          cache-dependency-path: readers-frontend/package-lock.json
+      - name: Install dependencies
+        run: |
+          cd readers-frontend
+          npm ci
+      - name: Install Playwright Browsers
+        run: |
+          cd readers-frontend
+          npx playwright install --with-deps
+      - name: Set up MySQL
+        run: |
+          sudo /etc/init.d/mysql start
+          mysql -e 'CREATE DATABASE test;' -uroot -proot
+          mysql -e 'SET GLOBAL SQL_MODE='ALLOW_INVALID_DATES';' -uroot -proot
+      - name: Run Playwright tests
+        run: |
+          cd readers-frontend
+          npm run test
+        env:
+          DATABASE_URL: "mysql://root:root@localhost:3306/test"
+          fileStoreUrl: "https://bbcdn.nouse.co.uk/file"
```

Comment:
Missing https://github.com/yorknouse/website/blob/astro/readers-frontend/src/components/utils/articles.ts#L56 ?

Reply:
Good catch! I've added it now